slides/src/content/unit-testing.md

```diff
@@ -115,9 +126,11 @@ describe('CapitalizePipe', () => {
 <!-- .slide: id="unit-testing-add-a-test" -->
 ## Add a Test
 
+- Use `fdescribe` to tell Jasmine to "**f**ocus" on this test -- we have several broken tests that we'll ignore for now.
```

Comment:
It's one thought, so I've reworded it.

I don't know if Karma/Jasmine has a filter feature that let's you select tests while in watch most like jest does.  Added issue #498 for this.